examples/basics/export_data.ipynb

```diff
@@ -1,18 +1,18 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.      # "batchId" "batch_id"
</pre><p>Please update to match the correct key, value format --&gt;  "batch_id" : "&lt;batch_id&gt;"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>